src/renderer/binary.ts

```diff
@@ -233,8 +235,8 @@ async function download(appState: AppState, version: string): Promise<string> {
  * @param {string} version
  * @returns {string}
  */
-function getDownloadPath(version: string): string {
-  return path.join(USER_DATA_PATH, 'electron-bin', version);
+function getDownloadPath(downloadPath: string, version: string): string {
```

Comment:
Is there a case where we don't want to use `appState.downloadBinaryPath` for the `downloadPath` param? Can we just hardcode the variable inside `path.join`?